README.md

```diff
@@ -111,29 +140,23 @@ This means that something is wrong with the ROM's contents. Either the baserom f
 Running `make init` will also make the `./expected` directory and copy all of the files there, which will be useful when running the diff script. The diff script is useful in decompiling functions and can be ran with this command: `./tools/asm-differ/diff.py -wmo3 <insert_function_here>`
 
 **Note**: to speed up the build, you can either:
-* Pass `-jN` to `make setup` and `make`, where N is the number of threads to use in the build, e.g. `make -j4`. The generally-accepted wisdom is to use the number of virtual cores your computer has.
-* Pass `-j` to `make setup` and `make`, to use as many threads as possible, but beware that this can use too much memory on lower-end systems.
+* Pass `-jN` to `make setup` and `make`, where N is the number of threads to use in the build, e.g. `make -j4`. The generally-accepted wisdom is to use the number of virtual cores your computer has, which is the output of `nproc` (which should be installed as part of `coreutils`).
+* Pass `-j` to `make setup` and `make`, to use as many threads as possible, but beware that this can use too much memory on lower-end systems, and even high-end systems tend to be fastest with roughly `-j$(nproc)` threads.
```

Comment:
I don't think the new stuff you added in this 2 lines add much to the building process instructions. I suggest to just remove them.

Reply:
The first is useful because it tells people how to find out how many virtual cores they might have, since often it's different from the physical number, and they may not even know how to check that.

We wrote the second comment based on what we knew at the time. Now we have more information (from @rylieb and my testing on higher-end machines), we know that discouraging use of `-j` is worth it on a wider range of machines. Indeed, if it was up to me, I would just remove the second bullet entirely.

Reply:
Yeah, I think the second bullet should go.